Neos.Eel/Tests/Unit/Helper/DateHelperTest.php

```diff
@@ -52,7 +53,7 @@ public function formatExamples()
         return [
             'DateTime object' => [$dateTime, 'Y-m-d H:i:s', '2013-07-03 12:34:56'],
             'timestamp as integer' => [1372856513, 'Y-m-d', '2013-07-03'],
-            'timestamp as string' => ['1372856513', 'Y-m-d', '2013-07-03'],
+            'timestamp as string' => ['@1372856513', 'Y-m-d', '2013-07-03'],
```

Comment:
The `$time` argument to `DateTime::__construct()` **is a string** and to use a unix timestamp one has to use `@1234567`. `date()` on the other hand only accepts integers. 

So what we have here is a case of _our own syntactic sugar broke a defined behaviour for the sake of making things "easier"_…

IMHO whenever I have a timestamp it is a proper integer, so this change would not affect me. But I like the `is_numeric` suhggestion…